parts/components/RotaryEncoder.cpp

```diff
@@ -21,11 +21,12 @@
 	along with MK404.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-
 #include "RotaryEncoder.h"
+#include "KeyController.h"
```

Comment:
Clang-tidy expects them to be sorted alphabetically, you have `K` after `R`

Reply:
I had `K` before `R` and it didn't like that either.